### ISSUE TITLE
"clip_grad_norm_" is necessary during training? 

### ISSUE THREAD

**Issue by utc1205** [zhangyp15/MonoFlex#26]
Hello ,
I see you use gradient clipping in training code, I wonder if it's necessary ? and  the gradient will be vanishing if without  gradient clipping ?
